tracer/test/snapshots/Elasticsearch5Tests.SubmitsTraces_SchemaV0.verified.txt

```diff
@@ -0,0 +1,3018 @@
+      span.kind: client,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_129,
+    SpanId: Id_130,
+    Name: elasticsearch.query,
+    Resource: Create,
+    Service: Samples.Elasticsearch-elasticsearch,
+    Type: elasticsearch,
+    Tags: {
+      component: elasticsearch-net,
+      elasticsearch.action: Create,
+      elasticsearch.method: PUT,
+      elasticsearch.url: http://localhost:00000/elastic-net-example/post/2/_create,
+      env: integration_tests,
+      out.host: elasticsearch,
+      runtime-id: Guid_1,
+      span.kind: client,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_131,
+    SpanId: Id_132,
+    Name: elasticsearch.query,
+    Resource: Create,
+    Service: Samples.Elasticsearch-elasticsearch,
+    Type: elasticsearch,
+    Tags: {
+      component: elasticsearch-net,
+      elasticsearch.action: Create,
+      elasticsearch.method: PUT,
+      elasticsearch.url: http://localhost:00000/elastic-net-example/post/2/_create,
+      env: integration_tests,
+      out.host: elasticsearch,
+      runtime-id: Guid_1,
+      span.kind: client,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_133,
+    SpanId: Id_134,
+    Name: elasticsearch.query,
+    Resource: CreateIndex,
+    Service: Samples.Elasticsearch-elasticsearch,
+    Type: elasticsearch,
+    Tags: {
+      component: elasticsearch-net,
+      elasticsearch.action: CreateIndex,
+      elasticsearch.method: PUT,
+      elasticsearch.url: http://localhost:00000/test_index_1,
+      env: integration_tests,
+      out.host: elasticsearch,
+      runtime-id: Guid_1,
+      span.kind: client,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_135,
+    SpanId: Id_136,
+    Name: elasticsearch.query,
+    Resource: CreateIndex,
+    Service: Samples.Elasticsearch-elasticsearch,
+    Type: elasticsearch,
+    Tags: {
+      component: elasticsearch-net,
+      elasticsearch.action: CreateIndex,
+      elasticsearch.method: PUT,
+      elasticsearch.url: http://localhost:00000/test_index_4,
+      env: integration_tests,
+      out.host: elasticsearch,
+      runtime-id: Guid_1,
+      span.kind: client,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
```

Comment:
You _may_ need to add an extra explicit sort order here to avoid flake related to flipping order, as these have the same resource names etc. Adding a `ThenBy` on the `elasticsearch.url` tag should work